hotel-search-api/src/test/java/com/hotel/api/search/mappers/RequestContextMapperTest.java

```diff
@@ -11,9 +12,14 @@ public class RequestContextMapperTest {
 
     private RequestContextMapper requestContextMapper;
 
+    private SupplierCredential supplierCredential = new SupplierCredential();
+
```

Comment:
if possible move this initialisation to setUp() method

Reply:
Moved it to setUp part